Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.yml

```diff
@@ -1324,7 +1324,7 @@ script:
         - contextPath: Bitbucket.PullRequest.source.repository.full_name
           description: The full name of the repository of the source branch.
           type: String
-        - contextPath: Bitbucket.PullRequest.source.repository.links.self.href
+        - contextPath: Bitbucket.PullRequest.source.repository.links
```

Comment:
You'll need to revert this. This is breaking backwards.
Add the new path that you want instead

Reply:
And have two contextPaths? 
It is supposed to be without .self.href, I probably missed it before

Reply:
Yes, we will have 2. one with self.href and one only with links